packages/hop-node/src/db/TransferRootsDb.ts

```diff
@@ -1,41 +1,41 @@
-import BaseDb, { KeyFilter } from './BaseDb'
+import BaseDb, { KV, KeyFilter } from './BaseDb'
 import chainIdToSlug from 'src/utils/chainIdToSlug'
 import { BigNumber } from 'ethers'
 import { Chain, ChallengePeriodMs, OneHourMs, OneWeekMs, RootSetSettleDelayMs, TxRetryDelayMs } from 'src/constants'
 import { normalizeDbItem } from './utils'
 import { oruChains } from 'src/config'
 
 interface BaseTransferRoot {
-  transferRootHash?: string
-  totalAmount?: BigNumber
-  destinationChainId?: number
-  sourceChainId?: number
-  sentCommitTxAt?: number
+  allSettled?: boolean
```

Comment:
nitpick: this alphabetization is awesome. can we do the same with `TransfersDb`.

Reply:
Will do